armi/nucDirectory/tests/test_elements.py

```diff
@@ -66,8 +63,22 @@ def test_element_addedElementAppearsInElementList(self):
         self.assertIn("BZ", elements.bySymbol)
         # re-initialize the elements
         with mockRunLogs.BufferLog():
-            elements.destroy()
+            nuclideBases.destroyGlobalNuclides()
             elements.factory()
+            nuclideBases.factory()
+            # Ensure that the burn chain data is initialized after clearing
+            # out the nuclide data and reinitializing it.
+            nuclideBases.burnChainImposed = False
+            with open(os.path.join(RES, "burn-chain.yaml"), "r") as burnChainStream:
+                nuclideBases.imposeBurnChain(burnChainStream)
```

Comment:
Yeah... since this test updates the global elements by adding "bacon" it will mess with later tests that check for the number of defined elements if the globals are not reset back to the original state. It's a quirk of nuclide and element data being global after ARMI is imported.